redash/query_runner/mongodb.py

```diff
@@ -251,7 +251,6 @@ def run_query(self, query, user):
                     step["$sort"] = SON(sort_list)
 
         if not aggregate:
-            s = None
```

Comment:
It looks like this whole if block can be removed, as it being overridden below.

Reply:
that's exactly. thanks!